DEBUG:g.py.<module> - Loaded roads: [[-1001790695108, 2000000172], [-791376008, 2000000066], [-602639062, -141127580], [-785949970, 2000000173]]
DEBUG:network.py._init - network initialized
INFO:main.py.<module> - Program is running!
DEBUG:vk_proc.py.single_vk_update - [219269, 532497, 2000000173, 1637178430, 'Mem-acho', '', {'attach1_type': 'photo', 'attach1': '-141127580_457239053', 'attach2_type': 'photo', 'attach2': '-141127580_457239054', 'from': '-141127580'}]
DEBUG:vk_proc.py.single_vk_update - *1000100000*
DEBUG:vk_proc.py.single_vk_update - Got message with attachments
DEBUG:vk_proc.py.single_vk_update - [219270, 532497, 2000000173, 1637178513, 'Mem-acho', 'HI mate bro man yuuoohoooo!!', {'attach1_type': 'photo', 'attach1': '-141127580_457239055', 'attach2_type': 'photo', 'attach2': '-141127580_457239057', 'from': '-141127580'}]
DEBUG:vk_proc.py.single_vk_update - *1000100000*
DEBUG:vk_proc.py.single_vk_update - Got message with attachments
